ctapipe/instrument/camera/tests/test_geometry.py

```diff
@@ -2,9 +2,30 @@
 import numpy as np
 from astropy import units as u
 from ctapipe.instrument import CameraGeometry, PixelShape
+from ctapipe.image.toymodel import Gaussian
 import pytest
 
 
+def create_mock_image(geom):
```

Comment:
Is this function superfluous? It is the same as below and according to codecov not covered by tests

Reply:
It is.
I moved the image conversion tests and forgot to remove this as well